packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -1568,6 +1649,8 @@ class EditableTextState extends State<EditableText> with AutomaticKeepAliveClien
       }
     }
 
+    // TODO(LongCatIsLooong): Update the text input configuration for non-web
+    // platforms.
```

Comment:
Is this something that needs a Github issue?

Reply:
I didn't give it too much thought, just felt we might need to update the configuration as well. Removing the TODO and taking a closer look.